pramen/core/src/main/scala/za/co/absa/pramen/core/reader/TableReaderJdbc.scala

```diff
@@ -122,6 +124,10 @@ class TableReaderJdbc(tableName: String,
     log.info(s"JDBC Query: $sql")
     val qry = sqlGen.getDtable(sql)
 
+    if (log.isDebugEnabled) {
```

Comment:
Curios, why the check and not just log.debug?

Reply:
This is a performance recommendation for debug logging. Generating a log message requires some CPU time and memory since string interpolation is used. Checking a boolean is much faster.